src/common/playlists/PlayHeaderActions.jsx

```diff
@@ -1,76 +1,159 @@
-import React, { useState } from "react";
+import React, { useCallback, useLayoutEffect, useState } from "react";
+import { useAuthenticated, useUserId } from "@nhost/react";
 import { BsGithub } from "react-icons/bs";
 import { IoLogoYoutube } from "react-icons/io";
 import { AiOutlineRead } from "react-icons/ai";
 import { BiComment } from "react-icons/bi";
 import { MdClose } from "react-icons/md";
-
+import Like from "common/components/Like/Like";
+import SignInMethods from "./SignInMethods";
 import Comment from "common/components/Comment";
+import MuiModal from "common/modal/MuiModal";
+import useLikePlays from "common/hooks/useLikePlays";
+import { NHOST } from "common/const";
+
+// we cannot remove delete SignInMethods, MuiModal file since in future we will support
```

Comment:
We can add it back when we plan for it, and probably the mechanism would be different. So my suggestion to remove them.

Reply:
removed